tools/update_fake_backends.py

```diff
@@ -49,17 +53,221 @@ def default(self, o):
+
+    @property
+    def rate_table(self):
+        """[cached] build rate_table information"""
+        if self._rate_table is None:
+            self._rate_table = []
+            total_readout_rate = self.properties.total_readout_rate
+            total_readout_rate_was = self.current_properties.total_readout_rate
+
+            if total_readout_rate[0] != 0 or total_readout_rate_was[0] != 0:
+                self._rate_table.append(
+                    [
+                        "readout error",
+                        _Summary._format_rate(total_readout_rate),
+                        _Summary._format_rate(total_readout_rate_was),
+                    ]
+                )
+            new_gates = self.properties.gate_errors.keys()
+            cur_gates = self.current_properties.gate_errors.keys()
+
+            for gate in set(new_gates).union(cur_gates):
+                rate = self.properties.gate_rate(gate)
+                rate_was = self.current_properties.gate_rate(gate)
+                self._rate_table.append(
+                    [
+                        f"{gate} error",
+                        _Summary._format_rate(rate),
+                        _Summary._format_rate(rate_was),
+                    ]
+                )
+            total_gate_rate = self.properties.total_gate_rate
+            total_gate_rate_was = self.current_properties.total_gate_rate
+            if total_gate_rate[0] != 0 or total_gate_rate_was[0] != 0:
+                self._rate_table.append(
+                    [
+                        "all gates",
+                        _Summary._format_rate(total_gate_rate),
+                        _Summary._format_rate(total_gate_rate_was),
+                    ]
+                )
+        return self._rate_table
+
+    @property
+    def gate_summary(self):
+        """[cached] the gate error == 1 summary"""
+        if self._gate_summary is None:
+            self._gate_summary = []
+            for gate, qubits in self.properties.gate_errors.items():
+                self._gate_summary.extend(
+                    [f" * {gate} {', '.join(map(str,qubit))}" for qubit in qubits]
+                )
+        return self._gate_summary
+
+    def generate_summary(self):
+        """A list with lines of the summary output"""
+        result = [f"\n{self.name}", "=" * len(self.name)]
+        output = []
+        if self.readout_summary:
+            output.append("Readout errors == 1:")
+            output += [f"{line}" for line in self.readout_summary]
+
+        if self.gate_summary:
+            output.append("Gate errors == 1:")
+            output += [f"{line}" for line in self.gate_summary]
+
+        if output:
+            result += output
+        else:
+            result += [self.props_msg]
+
+        if self.rate_table:
+            result.append(
+                tabulate(
+                    self.rate_table,
+                    [
+                        "error == 1",
```

Comment:
It might pay to be a bit more descriptive in the output here.  It's not immediately clear to me what `error == 1` means - I remember from the context of other PRs, but I wouldn't back myself to remember indefinitely, and new people using the script may well not know.  Maybe calling the headings "broken readouts" and "broken gates" (and the table heading something like "broken operations") could be a bit clearer.

In a similar vein, it wouldn't hurt to give this script more complete documentation (either as comments, or perhaps better as its docstring) explaining what's being measured for the summary output.